WebdriverIO.md

```diff
@@ -182,10 +185,27 @@ var openTopic = async function(topicName) {
 }
 ```
 
+As you may have noted above that `$$` selector is used by calling a function beacause we need to use `await` prefix with `$$` selector and we cannot place `await $$` directly at top as it will try to find the element as soon as the test will run, and this will cause the test to fail.
```

Comment:
Can you please also add the consequences of this sentence? From this line, my understanding is that it's good only right if the all selector element is found first when the test will run. But my question is what's the difference between declaring it at the top using await and then accessing it using the defined variable in comparision to declaring a function at the top and then calling the function in the tests in order to access the element.

`it will try to find the element as soon as the test will run`

Reply:
Updated the comment PTAL.